globals.go

```diff
@@ -625,6 +627,7 @@ const (
 	MergeTestRequester          = "merge_test"           // Evergreen commit queue
 	AdHocRequester              = "ad_hoc"               // periodic build
 	GithubMergeRequester        = "github_merge_request" // GitHub merge queue
+	AutoResetRequester          = "automatic_reset"
```

Comment:
I think this might not be the right place for this; requesters are meant to associate with types of versions. Seems like this is more a different kind of activator, so might live better here:
https://github.com/evergreen-ci/evergreen/blob/a0dbc9709914466344b06d1c3d04d341b7d142c6/globals.go#L234

Reply:
Sounds good, moved!